### PR TITLE
Dini Mueter...

### PR DESCRIPTION
**Pull request by dbrgn**
HANI MUETTER GHÖRT?

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -56,13 +56,20 @@ bot.on('message', function(message) {
...
+
+    // monkey island
     var monkey_island_match = monkey_island.search(message.text);
     if (monkey_island_match) {
       bot.sendMessage(message.chat.id, monkey_island_match, {'reply_to_message_id': message.message_id});
       return;
+
+    // mueter
+    } else if (message.text.search(/mue?tt?(er|i)/) != -1) {
+      bot.sendMessage(message.chat.id, 'HANI MUETTER GHÖRT?', {'reply_to_message_id': message.message_id});
```
Comment (dns2utf8) [inline bot.js]: missing H in 'GHÖHRT'

**Comment by rnestler** [comment]
This one will need a rebase after fixing the typo

File: bot.js (hunk 1 above)
Comment (dbrgn) [inline bot.js]: disagree ;)

**Comment by dbrgn** [comment]
Fixed and rebased!

**Comment by rnestler** [comment]
You could make it case insensitive.

File: bot.js (hunk 1 above)
Comment (rnestler) [inline bot.js]: "Ghört" is fine by me :smile:

**Comment by dbrgn** [comment]
@rnestler you just pushed a bad solution into my pull request ;) JavaScript has a RegEx modifier for that. I'll overwrite.

**Comment by dbrgn** [comment]
Yeah, and now deploy, I want to be insulted :laughing:

**Comment by rnestler** [comment]
I have an instance running at https://telegram.me/rnestler_flame_bot

File: bot.js [hunk 2]
```diff
@@ -70,6 +70,8 @@ bot.on('message', function(message) {
     var monkey_island_match = monkey_island.search(message.text);
     if (monkey_island_match) {
       replyText(monkey_island_match, message);
+    } else if (message.text.search(/mue?tt?(er|i)/i) != -1) {
```
Comment (netzhuffle) [inline bot.js]: Please use `/mue?tt?(er|i)/i.test(message.text)` as on line 75 for more consistent code.

**Comment by dbrgn** [comment]
Done.

**Comment by netzhuffle** [comment]
Thanks!

**Pull request merged**